tests/test_versioning.py

```diff
@@ -19,7 +19,7 @@ def test_route(app, handler):
 
 
 def test_bp(app, handler):
-    bp = Blueprint(__file__, version=1)
+    bp = Blueprint(__name__, version=1)
```

Comment:
Do we not want fixed names in these tests like in all others + examples? There's a few more like this in this specific file, but have just marked the first one.